models/cam-fv/readme.rst

```diff
@@ -165,16 +171,16 @@ which contain the types of observations in the table below
 ("T" = temperature, "U" = zonal wind, "V" = meridional wind,
 "Q" = specific humidity, "refractivity" = the bending of light by density variations).
 These files are available on NSF NCAR's glade file system:
-/glade/p/cisl/dares/Observations/NCEP+ACARS+GPS+AIRS/Thinned_x9x10.
+/glade/campaign/cisl/fs1_p_relocation/dares/Observations/NCEP+ACARS+GPS+AIRS/Thinned_x9x10.
 Versions of these files, which also have the results of the reanalysis in them,
-are available from the RDA ds345.0 linked above.
+are available from the RDA d345000 linked above.
```

Comment:
why not link here?

Reply:
When I first wrote it they were closer, and the extra typing didn't seem worthwhile.  I've made it a link now.